apps/server/src/modules/files-storage/entity/filerecord.entity.ts

```diff
@@ -1,8 +1,8 @@
 import { Embeddable, Embedded, Entity, Enum, Index, Property } from '@mikro-orm/core';
 import { ObjectId } from '@mikro-orm/mongodb';
 import { v4 as uuid } from 'uuid';
-import type { EntityId } from '../types/entity-id';
-import { BaseEntity } from './base.entity';
+import type { EntityId } from '../../../shared/domain/types/entity-id';
+import { BaseEntity } from '../../../shared/domain/entity/base.entity';
```

Comment:
Leads to cyclic import

Reply:
How come?

Reply:
Filerecord is importing from shared. And shared is importing filerecord.

Reply:
We run again and again in the cyclic import, but i think that we have some places that import is used instead of import type, or we use thinks from shared that we shouldn`t.